invenio/modules/collections/upgrades/collections_2015_05_28_recjson_tag_value.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Fix 'recjson_value' for 'collection idenfier' tag."""
+
+import warnings
+
+from invenio.ext.sqlalchemy import db
+
+depends_on = []
+
+
+def info():
+    """Info message."""
+    return __doc__
+
+
+def do_upgrade():
+    """Implement your upgrades here."""
+    name = 'collection identifier'
+    value = ''
+    recjson_value = '_collections'
+
+    tag = list(db.engine.execute(
+        """SELECT value, recjson_value FROM tag WHERE name=%s""", (name, )
+    ))
+
+    if not tag:
+        warnings.warn("Missing 'collection identifier' tag.")
+    elif not (tag[0][0] == value and tag[0][1] == recjson_value):
+        db.engine.execute(
+            """UPDATE tag SET value=%s, recjson_value=%s """
+            """WHERE name=%s""", (value, recjson_value, name))
+    else:
+        warnings.warn("'collection identifier' tag didn't require an upgrade.")
```

Comment:
Maybe add leading "Good, ..." to read "Good, collection identifier did not require..." so that people are not scared away by a "warning".